roles/mailserver/files/docker-compose.yml

```diff
@@ -138,9 +153,19 @@ services:
       - mailserver
       - postfixadmin
       - rainloop
+{% if www_is_opencart %}
+      - /mnt/docker/opencart:/opencart
+{% endif %}
```

Comment:
Is this set of lines in the wrong spot?
```yaml
{% if www_is_opencart %}
       - /mnt/docker/opencart:/opencart
{% endif %}
```

needs to be under volumes.

Does the nginx container even need to have a volume with visibility into the opencart container?